cms/templates/js/content-group-details.underscore

```diff
@@ -0,0 +1,13 @@
+<div class="wrapper-group-configuration">
```

Comment:
@frrrances - can we decide on a strategy for these `*-group-configuration` class names? It doesn't seem ideal that we're adding these to both group configurations and groups.